docs/howto/writing-migrations.txt

```diff
@@ -103,6 +103,8 @@ the respective field according to your needs.
     :filename: 0006_remove_uuid_null.py
 
     # -*- coding: utf-8 -*-
+    # Generated by Django 1.9
```

Comment:
The places where @tysonclugg added the comments make sense to me. They're in the context of "This is what your migration file looks like" and I'd like to keep that close to what Django generates.